### DISCUSSION TITLE
gopls cpu occupancy

### DISCUSSION THREAD

**Discussion by ghdwlsgur** [goreleaser/goreleaser#3395] [Q&A] [answered]
### What happened?

I used goreleaser to complete the deployment, and after completion I always saw gopls occupying a lot of cpu in a background process. Since it has been occupied for too long, I always directly down the goreleaser through the sudo kill pid command after distributing it. what's the problem in this case 😭

### How can we reproduce this?

Release is done through a bash script, but the script contents are as follows. 

export GITHUB_TOKEN=""
sudo rm -rf "$CURRENT"/dist "$CURRENT"/gopath
export GOPATH="$CURRENT"/gopath

 tag=$1
  if [ -z "$tag" ]; then
    echo "not found tag name"
    exit 1
  fi
 
  git tag -a "$tag" -m "Add $tag"
  git push origin "$tag"

  goreleaser release --rm-dist

### goreleaser version

```bash
goreleaser version 1.11.4
commit: d19ff6eb1eeca75e2bb3fbeaf473b3d0dc6ffc81
built by: homebrew
goos: darwin
goarch: amd64
```

### GoReleaser Check

- [X] goreleaser check shows no errors

### Search

- [X] I did search for other open and closed issues before opening this.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Additional context

It's a computer spec.

OS: macOS 12.5.1 21G83 x86_64
2018 MacBookPro 15
Intel i7-8850H (12) @ 2.60GHz
Intel UHD Graphics 630, Radeon Pro 560X
Memory: 9214MiB / 16384MiB

**Comment by caarlos0** [comment, marked as answer]
if anything, this seems to be a gopls bug... goreleaser does nothing to gopls directly, it does create a bunch of files in a `./dist` folder (which you should probably gitignore)...


I don't see what can be causing this, and never noticed this myself (and just tested, and could not reproduce), so I can't help much...
